### ISSUE TITLE
gh-pages demo

### ISSUE THREAD

**Issue by seiyria** [karkkant/ngBracket#2]
It'd be awesome to see what this is like before grabbing it and trying to get it to work.

**Comment by rgstephens** [comment]
I put up the demo code so you can give it a try here:

http://taxaidewa.org/ngbracket/demo.html

**Comment by seiyria** [comment]
Thanks. Ultimately I ended up making my own bracket tool (it was trivial enough) but having a project with their own stuff up on gh-pages is pretty important still.

**Comment by karkkant** [comment]
You're right, a live demo would be good. The whole demo should be rebuilt from scratch though, the old one is awfull in many ways and was made more for debugging than public demo.
